metric-schema-java/src/integrationInput/java/com/palantir/test/NamespaceTagsMetrics.java

```diff
@@ -55,8 +59,9 @@ public ProcessingBuilderResultStage processing() {
     public Meter more() {
         return registry.meter(MetricName.builder()
                 .safeName("namespace-tags.more")
-                .putAllSafeTags(tags)
                 .putSafeTags("locator", "package:identifier")
+                .putSafeTags("noValueTag", noValueTag)
+                .putSafeTags("locatorWithMultipleValues", locatorWithMultipleValues.getValue())
```

Comment:
Thoughts on extracting the string-value when we instantiate this utility object rather than each time a metric is accessed? That way all the namespaced tag fields use the `String` type.

Reply:
Sure

Reply:
DOne